datavault-broker/src/main/java/org/datavaultplatform/broker/controllers/PermissionsController.java

```diff
@@ -0,0 +1,101 @@
+package org.datavaultplatform.broker.controllers;
+
+import org.datavaultplatform.broker.services.PermissionsService;
+import org.datavaultplatform.common.model.PermissionModel;
+import org.datavaultplatform.common.model.RoleModel;
+import org.jsondoc.core.annotation.*;
+import org.jsondoc.core.pojo.ApiVerb;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/permissions")
+@Api(name="Permissions", description = "Interact with DataVault Roles and Permissions")
+public class PermissionsController {
+
+    private PermissionsService permissionsService;
+
+    public void setPermissionsService(PermissionsService permissionsService) {
+        this.permissionsService = permissionsService;
+    }
+
+    @ApiMethod(
+            path = "/permissions/role}",
```

Comment:
Stray closing brace here in the string literal (and the other @ApiMethod annotations).